.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   irb:
-    name: irb (${{ matrix.ruby }} / ${{ matrix.os }})
+    name: rake test ${{ matrix.ruby }} ${{ matrix.with_latest_reline && '(latest reline)' || '' }}
```

Comment:
We only run against `ubuntu-latest` so I think it's not necessary to indicate that in job names.